frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -793,6 +826,29 @@ pub mod pallet {
 			<MinimumUntrustedScore<T>>::set(maybe_next_score);
 			Ok(())
 		}
+
+		/// Set a solution in the queue, to be handed out to the client of this pallet in the next
+		/// call to `ElectionProvider::elect`.
+		///
+		/// This can only be set by `T::ForceOrigin`, and only when the phase is `Emergency`.
+		///
+		/// The solution is not checked for any feasibility and is assumed to be trustworthy, as any
+		/// feasibility check itself can in principle cause the election process to fail (due to
+		/// memory/weight constrains).
+		#[pallet::weight(T::DbWeight::get().reads_writes(1, 1))]
+		fn set_emergency_election_result(
+			origin: OriginFor<T>,
+			solution: ReadySolution<T::AccountId>,
+		) -> DispatchResult {
+			T::ForceOrigin::ensure_origin(origin)?;
+			ensure!(Self::current_phase().is_emergency(), <Error<T>>::CallNotAllowed);
+
+			// Note: we don't `rotate_round` at this point; the next call to
+			// `ElectionProvider::elect` will not succeed and take care of that.
+
+			<QueuedSolution<T>>::put(solution);
```

Comment:
Why not keep allowing solution submission?

We could ensure `Self::current_phase().is_unsigned_open() || Self::current_phase().is_emergency()` (instead of just the latter) in `fn unsigned_pre_dispatch_checks`. (Also the same for signed solution submission once we have that).

Keeping solution submission could provide a quicker solution than waiting for council to act

Reply:
Alternatively, what if we allowed `ForceOrigin` to flip the `Phase` as some sort of retry?

Reply:
If the unsigned OCW hasn't submitted a valid solution in the unsigned phase, it's unlikely to come up with a valid solution during the emergency phase. Allowing signed solutions during the emergency phase, subject to normal checks, sounds plausible to me, but that's out of scope of this PR.

I'm not sure what good it would do to manually flip the phase back to some other stage.

Reply:
> If the unsigned OCW hasn't submitted a valid solution in the unsigned phase, it's unlikely to come up with a valid solution during the emergency phase.

Good point, minimizing validator strain if we are already in an emergency sounds like a good heuristic

> Allowing signed solutions during the emergency phase, subject to normal checks, sounds plausible to me, but that's out of scope of this PR.

Agreed, sounds like an idea worth exploring once we introduce signed solutions 👍 

> I'm not sure what good it would do to manually flip the phase back to some other stage.

Could be an option in the back pocket for governance - but mostly thinking for signed phase and somehow skip unsigned